homeassistant/components/raincloud/binary_sensor.py

```diff
@@ -16,10 +16,13 @@
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType
 
-from . import BINARY_SENSORS, DATA_RAINCLOUD, ICON_MAP, RainCloudEntity
```

Comment:
I also moved shared constants to `const.py` and platform-specific constant to their respective module.